### PR TITLE
Add Localize() in client menus

### PR DESCRIPTION
**Pull request by awaysoft**

### CODE REVIEW

**Comment by nzyuzin** [comment]
Looks clean. :+1:

File: src/game/client/components/menus_settings.cpp [hunk 1]
```diff
@@ -1338,11 +1338,11 @@ void CMenus::RenderSettingsHUD(CUIRect MainView)
 
 		char name[16];
 		str_copy(name, g_Config.m_PlayerName, sizeof(name));
-		str_format(aBuf, sizeof(aBuf), "*** '%s' entered and joined the spectators", name);
+		str_format(aBuf, sizeof(aBuf), Localize("*** '%s' entered and joined the spectators"), name);
```
Comment (heinrich5991) [inline src/game/client/components/menus_settings.cpp]: These aren't actually translated in game, so I don't think it's appropiate to translate them in the setting dialog.

Comment (awaysoft) [inline src/game/client/components/menus_settings.cpp]: This is a demo line in settings -> HUD. Other demo lines are all translated.

Comment (heinrich5991) [inline src/game/client/components/menus_settings.cpp]: But these are the ones that appear untranslated in game. How would you translate them?

**Pull request closed without merging**